### PR TITLE
feat(sdk-coin-bera): add Berachain skeleton

### PR DESCRIPTION
**Pull request by hitansh-madan**
Ticket: WIN-490

### CODE REVIEW

File: modules/sdk-core/src/bitgo/environments.ts [hunk 1]
```diff
@@ -148,6 +150,7 @@ const testnetBase: EnvironmentTemplate = {
   seiNodeUrl: 'https://rest.atlantic-2.seinetwork.io',
   bldNodeUrl: 'https://devnet.api.agoric.net',
   zetaNodeUrl: 'https://rpc.ankr.com/http/zetachain_athens_testnet',
+  beraNodeUrl: '', // TODO: update url when live
```
Comment (Vijay-Jagannathan) [inline modules/sdk-core/src/bitgo/environments.ts:153]: testnet URL?

Comment (hitansh-bitgo) [reply modules/sdk-core/src/bitgo/environments.ts:153]: Testnet isn't live yet

File: modules/sdk-core/src/bitgo/environments.ts [hunk 2]
```diff
@@ -111,6 +112,7 @@ const mainnetBase: EnvironmentTemplate = {
   seiNodeUrl: 'https://rest.atlantic-2.seinetwork.io', //  TODO(BG-78997): Sei is still only in testnet update to mainnet url when it's live
   bldNodeUrl: 'https://agoric-api.polkachu.com',
   zetaNodeUrl: 'https://zetachain-athens.blockpi.network/lcd/v1/public', // TODO(WIN-142): update to mainnet url when it's live
+  beraNodeUrl: '', // TODO: update url when live
```
Comment (DinshawKothari) [inline modules/sdk-core/src/bitgo/environments.ts:115]: add a ticket number to this TODO.

File: modules/sdk-core/src/bitgo/environments.ts (hunk 1 above)
Comment (DinshawKothari) [reply modules/sdk-core/src/bitgo/environments.ts:153]: please add ticket number for TODO

**Review by DinshawKothari — DISMISSED** [review]

File: modules/sdk-core/src/bitgo/environments.ts (hunk 2 above)
Comment (hitansh-bitgo) [reply modules/sdk-core/src/bitgo/environments.ts]: added

File: modules/sdk-core/src/bitgo/environments.ts (hunk 1 above)
Comment (hitansh-bitgo) [reply modules/sdk-core/src/bitgo/environments.ts]: added

File: modules/account-lib/package.json [hunk 3]
```diff
@@ -31,6 +31,7 @@
     "@bitgo/sdk-coin-atom": "^3.6.11",
     "@bitgo/sdk-coin-avaxc": "^2.7.0",
     "@bitgo/sdk-coin-avaxp": "^3.7.11",
+    "@bitgo/sdk-coin-bera": "^1.0.0",
```
Comment (mmcshinsky-bitgo) [inline modules/account-lib/package.json:34]: Unless there is a specific use case for using this in account-lib elsewhere, let's remove it.

Comment (hitansh-bitgo) [reply modules/account-lib/package.json:34]: It's needed for wallet-platform, same for the other coins being added

**Review by alebusse — APPROVED** [review]
Reviewing changes outside of sdk-coin-bera

**Review by DinshawKothari — APPROVED** [review]

**Pull request merged**